charts/hivemq-platform/values.yaml

```diff
@@ -301,6 +301,14 @@ services:
     # The way HiveMQ platform authenticates client certificates. Possible values are `NONE`, `OPTIONAL` and `REQUIRED`
     tlsClientAuthenticationMode: "NONE"
 
+  # Legacy MQTT service configuration
```

Comment:
Should we have this in the `values.yaml` if we don't expose the migration switch here? Or should this just go into the migration documentation?

Reply:
@mhofsche What's your opinion on this?

Reply:
Lets keep this out for now. We make it part of the migration description.